docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md

```diff
@@ -45,7 +45,7 @@ This Source will be a Google Pub/Sub-only Source, which means that it will only
 
 1. **Advanced Options for Logs**.
 
-  ![GCP advanced options Jan 12 22.png](/img/send-data/GCP-advanced-options.png)
+  ![GCP advanced options Jan 12 22.png](/img/send-data/GCP-advanced-options.png|width=70)
```

Comment:
Needs to be in HTML. So it would be:

`<img src={useBaseUrl('img/send-data/GCP-advanced-options.png')} alt="GCP advanced options" width="70"/>`